internal/view/table.go

```diff
@@ -206,9 +207,11 @@ func (t *Table) cpCmd(evt *tcell.EventKey) *tcell.EventKey {
 	if path == "" {
 		return evt
 	}
-
 	_, n := client.Namespaced(path)
-	return cpCmd(t.app.Flash(), n)(evt)
+	v := tview.NewTextView().SetText(n)
```

Comment:
@mikutas I don't think we should build a view just to make the call. You can use the second helper function here since you have the text ie clipboardWrite. Does this make sense?